tiledb/api/src/array/attribute/arrow.rs

```diff
@@ -15,6 +15,20 @@ use crate::filter::arrow::FilterMetadata;
 use crate::filter::FilterListBuilder;
 use crate::{physical_type_go, Context, Datatype, Result as TileDBResult};
 
+// additional methods with arrow features
```

Comment:
nit: I'm not sure if we have guidelines around it, but for function or module-level comments I have a preference for block comments.

Reply:
Guidelines for idiomatic rust are to use the `//` on each line: https://doc.rust-lang.org/book/ch03-04-comments.html